component/src/main/java/org/wso2/extension/siddhi/io/http/util/ErrorHandlerUtils.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.wso2.extension.siddhi.io.http.util;
+
+import java.io.PrintStream;
+
+/**
+ * Class contains utility methods for ballerina server error handling.
+ */
+public class ErrorHandlerUtils {
+
+    private static final PrintStream outStream = System.err;
+
+    /**
+     * Print the error.
+     *
+     * @param throwable Throwable associated with the error
+     */
+    public static void printError(Throwable throwable) {
+        String errorMessage = throwable.getMessage();
+        if (errorMessage != null) {
+            outStream.println(errorMessage);
```

Comment:
Are we using this because logging is not initialized at this point? Also let's add the error cause as well. Can't we reuse this code at earlier occurrence?